src-backbone/app/js/models/session.js

```diff
@@ -126,7 +126,7 @@ let SessionModel = Backbone.Model.extend({
             },
             error: function(errors) {
                 App().RootView.clearNotifications();
-                let parsedErrors = JSON.parse(errors.responseText)
+                let parsedErrors = JSON.parse(errors.responseText);
```

Comment:
Leave this warning. I'm going to fix this and the account settings UI altogether later